<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" ignored="0" total="1" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="20789" started-at="2018-10-30T17:52:22Z" finished-at="2018-10-30T17:52:43Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="20789" started-at="2018-10-30T17:52:22Z" finished-at="2018-10-30T17:52:43Z">
      <class name="POM.TestCase.Signin_Errorhandling">
        <test-method status="PASS" signature="Invalidcrdentials()[pri:0, instance:POM.TestCase.Signin_Errorhandling@42d80b78]" name="Invalidcrdentials" duration-ms="20770" started-at="2018-10-30T17:52:22Z" finished-at="2018-10-30T17:52:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Invalidcrdentials -->
      </class> <!-- POM.TestCase.Signin_Errorhandling -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
